src/openforms/submissions/tests/test_variables/test_num_queries.py

```diff
@@ -129,7 +129,7 @@ def test_evaluate_form_logic_with_rules(self):
         # 7.  Delete submission attachment files
         # 8.  RELEASE SAVEPOINT
         # 9.  Delete submission values
-        with self.assertNumQueries(9):
+        with self.assertNumQueries(6):
```

Comment:
where's the explanation update of the queries being executed? The description above describes 9 queries, the assertion says 6.